src/dbt/kipptaf/models/extracts/python/rpt_python__manager_pm_averages.sql

```diff
@@ -15,8 +15,9 @@ with
             ) as score_measurement_shortname,
         from {{ ref("int_performance_management__observation_details") }}
         where
-            rubric_name = 'Coaching Tool: Coach ETR and Reflection'
+            rubric_name in ('Coach ETR', 'Coach ETR and Reflection: Part 1 - Scores')
             and row_score is not null
+            and academic_year >= 2023

```

Comment:
do we need the academic year filter?

Reply:
Yes. The rubric was different before 2023, so we can't compare it with older results